server/lib/activity-event.js

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (event, data, request, queryParams) {
+  data.op = 'activityEvent';
+  data.event = event;
+  data.userAgent = request.headers['user-agent'];
+
+  if (queryParams) {
+    queryParams.forEach(function (param) {
+      data[param] = request.query[param];
+    });
+  }
+
+  setImmediate(function () {
+    // The data pipeline listens on stderr.
+    process.stderr.write('activityEvent ' + JSON.stringify(data) + '\n');
```

Comment:
Yep. I'm aiming for total parity with activity events in the auth server, for simpler message matching in the data pipeline. The auth server does it like that, hence I'm doing it like that too.